### PR TITLE
latest ui changes

### PR DESCRIPTION
**Pull request by AnkRaiza**
validation

### CODE REVIEW

**Comment by jperata** [comment]
Add screenshots, since this is mostly front end changes.

File: src/components/Validation/ValidationParentComponent.tsx [hunk 1]
```diff
@@ -140,32 +140,39 @@ export class ValidationParentComponent extends React.Component<ValidationParentC
...
-                            <TooltipButton className={buttonStyle.info_button} icon={"info"} tooltip={"Enable notifications and get notified immediately whenever there are problems"} />
-                            {this.props && !this.props.vendorID && <TooltipButton className={buttonStyle.vendor_id_tooltip} onClick={redirectoToVendorIdpage} icon={"info"} tooltip={"To retrieve your vendor ID go to https://developer.amazon.com/mycid.html Please make sure it is for the correct organization if you belong to multiple."} />}
+                        <Cell style={{position: "relative"}} col={5} hideTablet={true} hidePhone={true}>
+                            <TooltipButton className={buttonStyle.info_button} icon={"info"} tooltip={"Enable Monitoring and get notified instantly when there is a problem with your voice app."} />
+                            {this.props && !this.props.vendorID && <TooltipButton className={buttonStyle.vendor_id_tooltip} onClick={redirectoToVendorIdpage} icon={"info"} tooltip={"Enter the vendor ID from your Amazon account at developer.amazon.com/mycid.html."} />}
```
Comment (jperata) [inline src/components/Validation/ValidationParentComponent.tsx]: do this link work without the https://

File: src/pages/NewSourcePage.tsx [hunk 2]
```diff
@@ -207,8 +202,7 @@ class CodeForm extends React.Component<CodeFormProps, CodeFormState> {
                         (
                             <Grid>
                                 <Cell col={12}>
-                                    <Button accent={true} raised={true} onClick={this.goToLogs}>Check for Logs</Button>
-                                    <Button style={{marginLeft: "30px"}} accent={true} raised={true} onClick={this.goToSettings}>Enable Monitoring</Button>
+                                    <Button accent={true} raised={true} onClick={this.goToLogs}>Contine</Button>
```
Comment (jperata) [inline src/pages/NewSourcePage.tsx]: Continue

File: src/components/Validation/ValidationParentComponent.tsx (hunk 1 above)
Comment (AnkRaiza) [reply src/components/Validation/ValidationParentComponent.tsx]: it's a tooltip, that is just the text on the box, the text is from what we agreed yesterday on product meeting

**Pull request merged**